packages/manager/modules/pci/src/projects/project/storages/databases/components/allowed-ip/allowed-ip.constants.js

```diff
@@ -0,0 +1,6 @@
+// reference from cloud-connect module
```

Comment:
This means we duplicate IPs Regex ? 😢

Reply:
Yeah... Probably, we can move it to a reusable later on...